app/controllers/api/v1/todos_controller.rb

```diff
@@ -12,6 +12,17 @@ def create
     end
   end
 
+  def index
+    user = User.find_by_auth_token(request.headers["Authorization"])
```

Comment:
Think about ways to refactor `TodosController` to remove this duplication, and how Devise is typically used to enforce authentication in a controller, and how Devise provides access to the logged in user once they've been authenticated.